### DISCUSSION TITLE
Is box-shadow on #79 working for anyone?

### DISCUSSION THREAD

**Discussion by Beowolve** [cssbattle/forum#60] [General]
I could get a much shorter version on target #79 using box-shadow, but I struggle with the typical 99.9% match problem. The image looks pixel perfect, but I can't get it to 100%. My working solution uses a different method but has some clear visual mismatches and still works.
This behavior is sometimes really nerve wrecking...
Does any top solution us box-shadow? Without getting into details I guess u guys know the technic I mean ;)

**Comment by SelenIT** [comment]
My solution uses box-shadows. But it took me dozens of unsuccessful attempts to find a combination that worked. In my case, it turned out `border-radius` value that was a tricky part (`5q` worked but neither `4q` and below nor `6q` and above did).

**Reply by Beowolve** [reply]
Thx @SelenIT that is good to know. Nevertheless, I think the scoring is buggy and the devs should look at it.

![image](https://user-images.githubusercontent.com/32326640/119883253-a060d200-bf2f-11eb-8c8b-d5de9528bee2.png)

These are the diff image for two solutions, where the 100% solution is less exact. The Images are generated with the preview tool, so this example should clearly show the problem. That target is just one of the problematic ones and for me that kind of takes the fun away :(.

@chinchang could you take a look at this?

**Reply by kushsolitary** [reply]
Hi @Beowolve, could you please DM us your solution on [Twitter](https://twitter.com/css_battle) so that we can take a look?

**Reply by Beowolve** [reply]
> Hi @Beowolve, could you please DM us your solution on [Twitter](https://twitter.com/css_battle) so that we can take a look?

Done,

And thx again @SelenIT, your tip really helped. I have different values but border-radius with q89 worked for me.
